src/HR2.py

```diff
@@ -0,0 +1,20 @@
+"""
+Task
+Given an integer,n, perform the following conditional actions:
+If n is odd, print Weird
+If nis even and in the inclusive range of 2 to 5, print Not Weird
+If n is even and in the inclusive range of 6 to 20, print Weird
+If n is even and greater than 20, print Not Weird
+"""
+
+n = int(input("Введите число:\n"))
+if n % 2 == 0 and 6 <= n <= 20:
+    print("Weird")
+elif n % 2 == 0 and 2 <= n <= 5:
+    print("Not Weird")
+elif n % 2 == 0 and n > 20:
+    print("Not Weird")
+elif n % 2 != 0:
+    print("Weird")
```

Comment:
`if n%2:1`
`if <n<:`
`elif`
(nested if)

Reply:
changed files in accordance with all comments.